Nautono/RTOS_framework_v1/src/main.c

```diff
@@ -9,12 +9,14 @@
 #include "sail_debug.h"
 #include "sail_tasksinit.h"
 #include "sail_pwm.h"
+#include "sail_motor.h"
+#include "sail_pwm.h"
 
 int main(void)
 {
 	CTRL_InitSystem(); // Init -> DEBUG UART, RADIO, EERPROM
 	//CTRL_InitSensors(); // Initialize the WeatherStation
```

Comment:
Is this comment accurate?

Reply:
In the current code, yes, but it will be updated to initialize more sensors. I think you made an update to sail_ctrl.c file but I didn't include any of those commits in this branch since I knew that commits made by me were working.